### PR TITLE
integrate suggestions from @roberge to /about page

### PR DESCRIPTION
**Pull request by bschmoker**

### CODE REVIEW

File: about/index.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ title: About STIX
 toc: about_toc.html
 ---
 
-Structured Threat Information Expression (STIX™) is a structured language to represent the full range of cyber threat information. The [STIX whitepaper](/getting-started/whitepaper) describes the motivation and architecture behind STIX, however at a high level the STIX language consists of 8 key constructs and the relationships between them:
+Structured Threat Information Expression (STIX™) is a structured language for describing cyber threat information so it can be shared, stored, and otherwise used in a consistent manner that facilitates automation.
```
Comment (johnwunder) [inline about/index.md]: @sbarnum likes to say "analyzed" here, so maybe "shared, stored, and analyzed in a consistent manner"

File: about/index.md [hunk 2]
```diff
@@ -19,9 +23,11 @@ Structured Threat Information Expression (STIX™) is a structured language to r
 
 <br class="clear: both;" />
 
-### The STIX Community
+## The STIX Community
+
+STIX is a collaborative effort that is led by DHS but driven by an open community. 
 
-STIX is a collaborative effort that is led by DHS but driven by an open community. Community members make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and supporting information. Join the community:
+Join the community to make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, and specifications.
```
Comment (johnwunder) [inline about/index.md]: maybe just "Join the community to make open-source contributions to the STIX language, tools, and documentation"? It reads kind of confusing now (mostly due to confusing language in the original version)

File: about/index.md [hunk 3]
```diff
@@ -29,23 +35,26 @@ STIX is a collaborative effort that is led by DHS but driven by an open communit
...
+## How do I get it?
 
 The STIX Language is available on the [STIX website](https://stix.mitre.org/language/).
 
 Bindings and related tools to help process and work with STIX are [open source on Github](https://github.com/STIXProject).
 
-#### Where can I find examples of STIX data? Are there any STIX repositories?
+## Where can I find examples of STIX data? Are there any STIX repositories?
+
+Certain STIX repositories [are available](/adopters) are available for free, while others are managed by paid services.
```
Comment (johnwunder) [inline about/index.md]: I like linking to the adopters page, but what does managed by paid services mean? Why not just say "while others are paid services".

File: about/index.md [hunk 4]
```diff
@@ -29,23 +35,26 @@ STIX is a collaborative effort that is led by DHS but driven by an open communit
...
 
-The STIX Samples page on this website hosts full threat reports expressed via STIX, including Mandiant's APT1 report and FireEye's Poison Ivy report. [Idioms](/idioms) also provide good constrained examples.
+[Example threat reports](/examples) are available for including Mandiant's APT1 report and FireEye's Poison Ivy report.
```
Comment (johnwunder) [inline about/index.md]: grammer, looks like there's an extra for in there and maybe a missing comma.

**Comment by johnwunder** [comment]
OK I added some inline comments. Some of them are nitpicky, sorry, the inline comments thing makes it too easy.

**Comment by bschmoker** [comment]
@johnwunder :+1: it's all good, will review

**Comment by bschmoker** [comment]
@roberge or @rroberge does this look okay?

File: about/index.md [hunk 5]
```diff
@@ -19,33 +23,40 @@ Structured Threat Information Expression (STIX™) is a structured language to r
...
+Individuals and organizations are invited to join the STIX community!
+
+Feature requests and bugfixes are managed using [open source repositories on Github](https://github.com/stixproject/schemas)
 
 - [Mailing List](https://stix.mitre.org/community/registration.html) — where community members discuss the latest drafts of the STIX schemas, specifications, utilities, technical documents, and other items integral to the ongoing development of STIX.
 - [Chat](https://gitter.im/STIXProject/schemas) - informal discussions and help
-- [Code Repositories](https://github.com/STIXProject/) — the central location for STIX Community members to make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and other supporting information and items.
+- [Code Repositories](https://github.com/STIXProject/) — Github organization for development of schemas, tools, specifications, and other supporting information and items.
```
Comment (johnwunder) [inline about/index.md]: I know it's nitpicky but can we change "Github organization" back to "the central location"? It's just much more clear and I don't think most people know what "Github organization really means.

File: about/index.md [hunk 6]
```diff
@@ -19,33 +23,40 @@ Structured Threat Information Expression (STIX™) is a structured language to r
...
 
-#### Where can I find examples of STIX data? Are there any STIX repositories?
+## Where can I find examples of STIX data? Are there any STIX repositories?
+
+Certain STIX repositories [are available](/adopters) are available for free and for-pay.
 
-The STIX Samples page on this website hosts full threat reports expressed via STIX, including Mandiant's APT1 report and FireEye's Poison Ivy report. [Idioms](/idioms) also provide good constrained examples.
+[Example threat reports](/examples) include STIX versions of Mandiant's APT1 report and FireEye's Poison Ivy report. 
 
-In addition to the MITRE samples, community members have set up [TAXII](https://taxii.mitre.org) repositories containing STIX content and even directories pointing to those repositories. One example repository is http://hailataxii.com.
```
Comment (johnwunder) [inline about/index.md]: IMO we should continue linking to hailataxii. People ask for STIX data all the time, it's the best source for that, and it's free.

**Comment by johnwunder** [comment]
I added two comments inline, otherwise this looks good.

**Comment by sbarnum** [comment]
How are you correctly rendering the site to do review?

sean

**Comment by johnwunder** [comment]
I just read the diff, mainly looking for content vs. presentation (I assume if they're submitting a PR it builds correctly).

**Comment by johnwunder** [comment]
That said I just merged this one and the headings are a bit funky. If you want to use `##` for all of the FAQ questions and effort names (which IMO is big and messy but I don't really care) then you should use `#` for the "Relationship to other efforts"

**Comment by bschmoker** [comment]
sorta confused, is this merged to master? whichever heading looks good is cool

**Comment by johnwunder** [comment]
Merged most of these separately

**Pull request closed without merging**